README-FIPS.md

```diff
@@ -77,6 +81,78 @@ If you configured OpenSSL to be installed to a different location, the paths wil
 vary accordingly. In the rare case that you need to install the fipsmodule.cnf
 to non-standard location, you can execute the `openssl fipsinstall` command manually.
 
+Installing the FIPS provider and using it with the latest release
+=================================================================
+
+This normally requires you to download 2 copies of the OpenSSL source code.
+
+Download and build the (OpenSSL 3.0) FIPS provider
+--------------------------------------------------
+
+OpenSSL 3.0 is the only source that is currently FIPS validated.
+If you use any other source currently it is NOT FIPS valid.
```

Comment:
`NOT FIPS valid` or `NOT FIPS validated`?

Reply:
The text has changed